src/modules/packages/packages.schema.yaml

```
@@ -33,7 +33,7 @@ properties:
         properties:
             num_retries: { type: integer, default: 0 }
             disable_download_timeout: { type: boolean, default: false }
-            needed_only: { type: boolean, default: false }
+            needed_only: { type: boolean, default: true }
```

```suggestion
            needed_only: { type: boolean, default: false }
```

leave as default it's set in the .conf file